tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot

```diff
@@ -199,7 +199,10 @@ Upgrade VCH with unreasonably short timeout and automatic rollback after failure
     ${rc}  ${output}=  Run And Return Rc And Output  govc snapshot.tree -vm=%{VCH-NAME}
     Should Be Empty  ${output}
     # the appliance is restarting - attempt to wait until it's ready
-    Sleep  90
+    # version of appliance we rolled back to is old, so we have to set DOCKER_API_VERSION
+    Set Environment Variable  DOCKER_API_VERSION  1.23
```

Comment:
do we need to unset `DOCKER_API_VERSION` once this test is run?

Reply:
Currently yes -- the version that it's rolled back to is old...so doesn't support the current client...

Reply:
Yes, agree with setting it but should it need to be reverted for subsequent tests? because its set as env variable or instead it can be set as test variable using `Set Test Variable` which applies to only specific test.

Reply:
_maybe_ ... as long as that `Test Variable` is available to the docker cli in linux...also I assumed the ENV would go away before the next test..group..suite...

certainly can change if it causes a problem...or your think it will..

Reply:
Yeah, I think test variable is available everywhere but applies only within that test scope unlike env variable which applies to all tests. Another option is to remove env variable using `Remove Environment Variable` as part of teardown.

Reply:
addressed: #7693

Reply:
Sorry, late on review but I see you already merged it but that remove keyword doesn't take value, so its just `This keyword doesn't take value, so its just `Remove Environment Variable DOCKER_API_VERSION`